CMakeLists.txt

```diff
@@ -176,9 +176,18 @@ foreach(api ${nidrivers})
   endif()
 endforeach()
 
+if(CMAKE_SYSTEM_NAME STREQUAL Linux)
+  set(OSDEP_SOURCE
+    "source/server/linux/syslog_logging.cpp")
+else()
+  set(OSDEP_SOURCE "")
+endif()
```

Comment:
I think an alternative would be to add the source conditionally with[ target_sources](https://cmake.org/cmake/help/latest/command/target_sources.html), but I've never tried that so feel free to leave as-is for now.
```
if(CMAKE_SYSTEM_NAME STREQUAL Linux)
   target_sources(ni_grpc_device_server
      PRIVATE "source/server/linux/syslog_logging.cpp")
endif()
```

Reply:
That worked great, thanks! I'm new at CMake, so it's been interesting to try it out.

Reply:
Glad it worked! I'm definitely still learning about it as we've refined things here.